src/components/UploadButton/UploadButton.test.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import UploadButton from './UploadButton';
+
+test('Confirms that math works', () => {
```

Comment:
Do you mean to commit the test as-is? I think we either fix it to actually test something or we remove it.

Reply:
Decided to toss it for now. Tests on this can be added in a new pull request.